SmartSaver.MVC/Views/Home/Index.cshtml

```diff
@@ -2,7 +2,40 @@
     ViewData["Title"] = "Home Page";
 }
 
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-</div>
+<html>
+@{var myStyleSheet = "~/css/generalDesign.css";}
```

Comment:
I believe that general styling could be written in ~/css/site.css (it is used in layout).